### PR TITLE
Document YKD_FORCE_TRACE_DECODER

### PR DESCRIPTION
**Pull request by vext01**
and one more small addition.

Requires https://github.com/ykjit/yk/pull/598 to be merged first.

### CODE REVIEW

File: src/dev/env.md [hunk 1]
```diff
@@ -7,8 +7,22 @@ Variables prefixed with `YKD_` are intended for debugging only. Most (if not
 all) of the debugging variables introduce extra computation that slows down
 program execution.
 
+Some variables are conditionally compiled in when the containing crate is built
```
Comment (ltratt) [inline src/dev/env.md:10]: Should we make clearer that this is an *internal* feature?

[Also s/compiled in/compiled/]

Comment (vext01) [reply src/dev/env.md]: Added prose on the internalness of it.

I meant `compiled-in`. Try [a208b12](https://github.com/ykjit/ykdocs/pull/35/commits/a208b12ddc88a74f797b750789fcda646f5806b9).

**Comment by vext01** [comment]
(I've sneaked in a few other fixes too)

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**